### PR TITLE
Add archetype

### PR DESCRIPTION
**Pull request by erdlet**
As mentioned in #162, I created an archetype for Jakarta EE projects with the MVC API and Krazo a few months ago. But I think, that this archetype is better placed in the Krazo repo, as it is released in sync with out newest versions then.

### CODE REVIEW

File: archetype/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+  ~ Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
```
[the remark this answers is not in the document]
Comment (erdlet) [reply archetype/pom.xml]: Oh, it seems that my copyright profile is outdated. Thanks for that catch, I'll change it!

File: archetype/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>war</packaging>
+
+    <name>${artifactId}</name>
+
+    <properties>
+        <jakartaee-api.version>8.0.0</jakartaee-api.version>
+        <krazo.version>1.0.0</krazo.version>
```
[the remark this answers is not in the document]
Comment (erdlet) [reply archetype/src/main/resources/archetype-resources/pom.xml]: Good catch, that needs to be updated

[the remark this answers is not in the document]
Comment (erdlet) [reply archetype/src/main/resources/archetype-resources/pom.xml]: Good catch, that needs to be updated

File: archetype/src/main/resources/archetype-resources/pom.xml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        <javax.mvc-api.version>1.0.0</javax.mvc-api.version>
```
[the remark this answers is not in the document]
Comment (erdlet) [reply archetype/src/main/resources/archetype-resources/pom.xml]: Great idea! It was a little bit tricky, but now it works :)

**Comment by chkal** [comment]
I just merged the PR. I guess it is time for 1.1.0-M2 soon. :laughing:

**Pull request merged**